account_invoice_rounding_by_currency/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Unit rounded invoice (Swedish rounding) by currency
+=======================================================
+
+This module extende functionallity of module `Unit rounded invoice <https://github.com/OCA/account-invoicing/tree/8.0/account_invoice_rounding>`_.
+
+Add a parameter to give a unit for rounding such as CHF 0.05 for Swiss
+invoices
+
```

Comment:
Please add 'configuration' section, like in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst